Code-the-Dream-School/dd-prac-team1-front
# Devips/loader

## One Line Description
Created loader

## Requirements

_What this code should do_

## Notes

_Any additional info_

## Test Steps
on route /ai-recipe try to generate new recipe. during waiting response you should see the loader. It should be disappear with API response.

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
